.github/workflows/manage-flux-releases.yml

```
@@ -0,0 +1,196 @@
+            }
+            // Create a new issue
+            const response = await github.rest.issues.create({
+              ...context.repo,
+              title: issueTitle,
+              body: issueBody,
+              labels: [ ${{env.LABEL}} ]
+            });
+            console.log(`Issue created: ${response.data.html_url}`);
+  create-release-pr:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues' && github.event.action == 'edited' && startsWith(github.event.issue.title, github.env.ISSUE_TITLE_PREFIX) && github.event.issue.draft == false && github.event.issue.state == 'open'
```

actionlint (1.7.7)

receiver of object dereference "issue_title_prefix" must be type of object but got "string"

(expression)